Sources/Logging/Strings/OfferingStrings.swift

```diff
@@ -169,3 +178,27 @@ extension OfferingStrings: LogMessage {
     var category: String { return "offering" }
 
 }
+
+
+extension Configuration.APIKeyValidationResult {
+
+    var storeNameForLogging: String? {
+        switch self {
+        case .validApplePlatform, .legacy:
+            return "App Store"
+        case .simulatedStore:
+            return "Test Store"
+        case .otherPlatforms:
+            return nil
+        }
+    }
+
+    var indefiniteArticle: String {
+        switch self {
+        case .validApplePlatform, .legacy:
+            return "an" // "an App Store API key"
+        case .otherPlatforms, .simulatedStore:
+            return "a" // "a Test Store API key"
+        }
+    }
```

Comment:
Chose to be practical on this and didn't want to get too fancy (I couldn't think of a more elegant, quick solution either). Lmk if you have other ideas!